tests/test_filter_pipeline_v2.py

```diff
@@ -0,0 +1,27 @@
+""" Unit tests for pyfive's ability to read a file with filter_pipeline v2
+(as is found in some new netCDF4 files) """
+import os
+import warnings
+
+import numpy as np
+from numpy.testing import assert_array_equal, assert_almost_equal
```

Comment:
Unused imports, remove

Reply:
@bmaranville 

> I tried for a while to create a new file using netCDF4 but was unable to trigger the switch to filter pipeline description v2.

That would be the way forward, I think.